src/DetailsView/components/failure-instance-panel-control.tsx

```diff
@@ -115,10 +130,73 @@ export class FailureInstancePanelControl extends React.Component<FailureInstance
         );
     }
 
+    protected getSelectorAndSnippet = (): JSX.Element => {
+        return (
+            <div>
+                <a className="learn-more"> Learn more about adding failure instances </a>
+                <TextField
+                    className="selector-failure-textfield"
+                    label="CSS Selector"
+                    multiline={true}
+                    rows={8}
+                    value={this.state.selector}
+                    onChange={this.onSelectorChange}
+                    resizable={false}
+                    defaultValue="CSS selector"
+                />
+                <div>Note: If the CSS selector maps to multiple snippets, the first will be selected.</div>
+                <div>
+                    <DefaultButton text="Validate CSS selector" onClick={this.onValidateSelector} disabled={this.state.selector === ''} />
+                </div>
+                <div>
+                    <label>Code Snippet</label>
+                    <div>{this.state.snippet}</div>
+                </div>
+            </div>
+        );
+    };
+
+    protected panelWithoutSelectorAndSnippet = (testStepConfig: Readonly<Requirement>, panelProps: GenericPanelProps): JSX.Element => {
```

Comment:
I didn't find any usage of this function (vscode shows "0 references" and I commented out the code and it still builds). 
If we're not using it, let's remove it!

Reply:
whoops! forgot to delete!